packages/web-pkg/src/components/sideBar/Details/SpaceDetailsMultiple.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <div class="oc-mt-xl">
+    <div class="oc-flex space-info">
+      <oc-icon name="layout-grid" size="xxlarge" />
+      <p v-translate>Multiple spaces selected</p>
```

Comment:
Please open a followup issue, the multi selected space panel could show some more info. E.g. how many spaces are selected and what their combined quota values are (total, remaining, used).